html/om_oss.html

```diff
@@ -19,29 +18,91 @@
                 <input id="search" type="text" placeholder="Sök">
             </form>
 
-            <ul>
-                <li><a href="html/om_oss.html">Om oss</a></li>
-                <li><a href="html/konstnarsguiden.html">Konstnärsguiden</a></li>
-                <li><a href="#">Vanliga frågor</a></li>
-                <li><a href="html/kontakt.html">Kontakt</a></li>
-                <li>Meny</li> <!-- Drop down menu here -->
-            </ul>
-
-            <a href="#"><img src="#" alt="svenska">Sv</a>
-            <a href="#"><img src="#" alt="english">En</a>
+        <ul>
+          <li><a href="html/om_oss.html">Om oss</a></li>
+          <li><a href="html/konstnarsguiden.html">Konstnärsguiden</a></li>
+          <li><a href="#">Vanliga frågor</a></li>
+          <li><a href="html/kontakt.html">Kontakt</a></li>
+          <li>Meny</li>
+          <!-- Drop down menu here -->
+        </ul>
 
-            <a href="#">Logga in</a>
+        <a href="#"><img src="#" alt="svenska" />Sv</a>
+        <a href="#"><img src="#" alt="english" />En</a>
 
-        </nav>
+        <a href="#">Logga in</a>
+      </nav>
     </header>
 
     <main>
```

Comment:
The breadcrumbs went missing here :)

Reply:
No idea what it was. I thought was a mistake and I deleted it 🤣

Reply:
Hahahaha please add them back 🤣🙏
With link to index.html on the first, the second one doesn't need to be a link. It can simply be a list item.